Cesium3DTilesSelection/CMakeLists.txt

```diff
@@ -25,16 +25,19 @@ target_sources(
     PRIVATE
         ${CESIUM_3DTILES_SELECTION_SOURCES}
         ${CESIUM_3DTILES_SELECTION_HEADERS}
-    PUBLIC
         ${CESIUM_3DTILES_SELECTION_PUBLIC_HEADERS}
 )
 
 cesium_target_include_directories(
     TARGET 
         Cesium3DTilesSelection
     PUBLIC
-        ${CMAKE_BINARY_DIR}
-        ${CMAKE_CURRENT_LIST_DIR}/include
+        $<BUILD_INTERFACE:${CESIUM_NATIVE_RAPIDJSON_INCLUDE_DIR}>
+        $<BUILD_INTERFACE:${CESIUM_NATIVE_DRACO_INCLUDE_DIR}>
+        $<BUILD_INTERFACE:${CESIUM_NATIVE_LIBMORTON_INCUDE_DIR}>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}>
```

Comment:
I think those one isn't new, but it's not obvious why it's needed, either. Do you happen to know?